apps/babydegen-ui/src/components/WithdrawAgentsFunds/useFunds.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { LOCAL } from '../../constants/urls';
+import { WithdrawalFunds } from '../../types';
+import { mockFunds } from '../../mocks/mockFundsWithdrawal';
+
+const IS_MOCK_ENABLED = process.env.IS_MOCK_ENABLED === 'true';
+
+export const useFunds = () =>
+  useQuery<WithdrawalFunds>({
+    queryKey: ['funds'],
```

Comment:
can we just in case use different naming, e.g. withdrawalFunds, I'm worried we accidentally use it for some other cases on the UI and it'll cause problems